public/js/modules.js

```diff
@@ -3457,7 +3457,7 @@ PriorityEncoder.prototype.customDraw = function () {
     for (var i = 0; i < this.bitWidth; i++) {
         fillText(ctx, String(i), xx + this.output1[0].x - 10, yy + this.output1[i].y + 2, 10);
     }
-    fillText(ctx, "EN", xx + this.enable.x, yy + this.enable.y - 5, 10);
```

Comment:
Ahh, we cannot rename `this.enable` actually as old circuits will stop working as they expect it to be `this.enable`. However, you can change the fillText() alone.